mne/datasets/utils.py

```diff
@@ -271,7 +274,8 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
                 'tar.gz/%s' % releases['testing'],
         multimodal='https://ndownloader.figshare.com/files/5999598',
         visual_92_categories='https://mne-tools.s3.amazonaws.com/datasets/%s',
-        mtrf="https://superb-dca2.dl.sourceforge.net/project/aespa/%s"
+        mtrf="https://superb-dca2.dl.sourceforge.net/project/aespa/%s",
+        fieldtrip_cmc="ftp://ftp.fieldtriptoolbox.org/pub/fieldtrip/tutorial/%s"
```

Comment:
please also modify `circle.yml` to autodetect when to download this dataset. You can see where the conditionals are for our other datasets